humble/distribution.yaml

```diff
@@ -3674,7 +3674,7 @@ repositories:
     release:
       tags:
         release: release/humble/{package}/{version}
-      url: https://github.com/ika-rwth-aachen/message_tf_frame_transformer-release.git
+      url: https://github.com/ros2-gbp/message_tf_frame_transformer-release.git
       version: 1.1.0-1
```

Comment:
Looking at https://github.com/ros2-gbp/message_tf_frame_transformer-release/blob/master/tracks.yaml, it is clear that a release of this package was done into the old release repository that is not reflected in the new one.  How do we want to handle this?

Reply:
What precisely do you mean by "not reflected"?

I specifically documented a discrepancy in the PR description:

> there is divergent bloom info between the ros2-gbp repository and the upstream one. But all tags, which are the important artifacts for release integrity, have been copied over to the ros2-gbp repository.

I'll double check whether the changes from the external release repository's bloom configuration need to be incorporated for the migration tool to work as intended before merging.

> Looking at https://github.com/ros2-gbp/message_tf_frame_transformer-release/blob/master/tracks.yaml, it is clear that a release of this package was done into the old release repository that is not reflected in the new one

Furthermore, it is _not_ clear to me what specifically you're trying to highlight as  a difference between the external tracks.yaml and the ros2-gbp one. A line-specific link or description of the discrepancy would be appreciated in the future.

Reply:
As previously stated, all tags were propagated so any past release artifacts are available in the ros2-gbp repository whether the release was originally performed using the external repository or the ros2-gbp one. I am fairly certain that the script would function correctly since it uses the rosdistro metadata as its primary source of truth, but that doesn't mean that future manual releases to the other ROS 2 distributions would behave as expected. So I went ahead and integrated the changes from the external repository's tracks.yaml in https://github.com/ros2-gbp/message_tf_frame_transformer-release/commit/3b498040041fe067ffa9cf9e496d50cfb385b108